### PR TITLE
baudline: init at 1.0.8

### PR DESCRIPTION
**Pull request by bjornfor**
Baudline is a scientific signal analysis application.

I was unsure whether the baudline license allowed packaging in nixpkgs,
so I sent a copy of this nix expression (and some explanation) to their
support. I got this reply:

  From: Erik Olson erik@sigblips.com
  Subject: Re: Licensing question

  Hello Bjorn,

  This sounds fine.  What you suggest doing is very similar to how the
  FreeBSD FreshPorts system deals with baudline.

  Erik

So basically, everything is good as long as we only distribute the _expression_
to install the package. We must not distribute the package (binary).

### CODE REVIEW

File: pkgs/applications/audio/baudline/default.nix [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+      files with a multitude of effects and filters.  The baudline signal
+      analyzer combines fast digital signal processing, versatile high speed
+      displays, and continuous capture tools for hunting down and studying
+      elusive signal characteristics.
+    '';
+    homepage = http://www.baudline.com/;
+    # See http://www.baudline.com/faq.html#licensing_terms
+    license = licenses.unfree;
+    platforms = [ "x86_64-linux" "i686-linux" ];
+    hydraPlatforms = []; # do NOT (re)distribute on hydra
```
Comment (vcunat) [inline pkgs/applications/audio/baudline/default.nix]: I don't think setting `hydraPlatforms` is needed. Hydra won't build unfree packages by default. It seems even a convention that `licenses.unfree` means non-redistributable, as we also have `unfreeRedistributable`.

File: pkgs/applications/audio/baudline/default.nix [hunk 2]
```diff
@@ -0,0 +1,70 @@
+{ stdenv, fetchurl, libXmu, libXt, libX11, libXext, libXxf86vm, jack
+, makeWrapper
+}:
+
+let
+  rpath = stdenv.lib.makeSearchPath "lib"
```
Comment (vcunat) [inline pkgs/applications/audio/baudline/default.nix]: Why not `stdenv.lib.makeLibraryPath` instead? (That would also fix it on the closure-size branch.)

**Comment by bjornfor** [comment]
@vcunat: Thanks for the pointers (I didn't know/think of that). Pushed updated commit.

**Comment by bjornfor** [comment]
Pushed to master (https://github.com/NixOS/nixpkgs/commit/e8c22b33e6ca0d4b298f0a7dcb4208c0cbeb3ddf).
